roles/ceph-mon/tasks/create_mds_filesystems.yml

```diff
@@ -1,9 +1,19 @@
 ---
 - name: create filesystem pools
-  command: "{{ docker_exec_cmd }} ceph --cluster {{ cluster }} osd pool create {{ item.name }} {{ item.pgs }}"
+  command: >
+    {{ docker_exec_client_cmd }} --cluster {{ cluster }}
```

Comment:
s/docker_exec_client_cmd/docker_exec__cmd ?
I guess this is because of a copy/paste ?